test/gleam/http_test.gleam

```diff
@@ -722,6 +745,195 @@ pub fn get_query_test() {
   should.equal(Error(Nil), http.get_query(request))
 }
 
+pub fn set_query_test() {
+  let request = http.Request(
+    method: http.Get,
+    headers: [],
+    body: Nil,
+    scheme: http.Https,
+    host: "example.com",
+    port: None,
+    path: "/",
+    query: None,
+  )
+
+  let query = [tuple("answer", "42"), tuple("test", "123")]
+  let updated_request = http.set_query(request, query)
+  updated_request.query
+  |> should.equal(Some("answer=42&test=123"))
+
+  let empty_query = []
+  let updated_request = http.set_query(request, empty_query)
+  updated_request.query
+  |> should.equal(Some(""))
+}
+
+pub fn get_req_header_test() {
+  let make_request = fn(headers) {
+    http.Request(
+      method: http.Get,
+      headers: headers,
+      body: Nil,
+      scheme: http.Https,
+      host: "example.com",
+      port: None,
+      path: "/",
+      query: None,
+    )
+  }
+
+  let header_key = "GLEAM"
+  let request = make_request([tuple("answer", "42"), tuple("gleam", "awesome")])
+  should.equal(
+    Ok("awesome"),
+    request
+    |> http.get_req_header(header_key))
+
+  let request = make_request([tuple("answer", "42")])
+  should.equal(
+    Error(Nil),
+    request
+    |> http.get_req_header(header_key))
+}
```

Comment:
Could we write these assertions like this:

```
request
|> http.get_req_header(header_key)
|> should.equal(Error(Nil))
```

I think it is easier to read, and the "expected" value is intended to be the second argument.

Reply:
Definitely, that makes sense. Updated to reflect this change.